FEXCore/Source/Interface/IR/Passes/RegisterAllocationPass.cpp

```diff
@@ -461,9 +466,11 @@ void ConstrainedRAPass::Run(IREmitter* IREmit_) {
   auto IR_ = IREmit->ViewIR();
   IR = &IR_;
 
+  // SSAToNewSSA, NewSSAToSSA allocated on first-use
+  SSAToNewSSA.clear();
+  NewSSAToSSA.clear();
```

Comment:
Wouldn't it make more sense to clear these at the end of `Run()` rather than at the beginning of its next invocation?

The underlying storage won't be freed either way, but it's easier to reason about code when there are clear invariants like "SSAToNewSSA is empty outside of Run()".